packages/jbrowse-plugin-apollo/src/ApolloInternetAccount/components/ApolloLoginForm.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  TextField,
+} from '@material-ui/core'
+import React, { useState } from 'react'
+
+export const ApolloLoginForm = ({
+  authenticationURL,
+  internetAccountId,
+  handleClose,
+}: {
+  authenticationURL: string
+  internetAccountId: string
+  handleClose: (arg?: string) => void
+}) => {
+  const [username, setUsername] = useState('')
+  const [password, setPassword] = useState('')
+
+  async function onSubmit(
+    this: unknown,
+    event: React.FormEvent<HTMLFormElement>,
+  ) {
```

Comment:
The unused `this` parameter isn't necessary, and it's probably best to call `event.preventDefault()` at the top of the function to make sure it runs.

```suggestion
  async function onSubmit(event: React.FormEvent<HTMLFormElement>) {
    event.preventDefault()
```